src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/CommandMap.java

```diff
@@ -0,0 +1,30 @@
+package alvioneurope.gsergeev.javagroup.clientlogimporter;
+
+import java.util.*;
+
+/**
+ * Created by GSergeev on 11/16/2015.
+ */
+public class CommandMap {
+    private Map<Integer,String> commandMap;
+
+    public CommandMap() {
+        commandMap = new TreeMap<Integer, String>();
```

Comment:
I'm not shure that TreeMap is the best data structure for this case. Let discuss it. Please analize http://habrahabr.ru/post/237043/